README.md

```diff
@@ -305,4 +348,21 @@ associative array in the first argument:
 </div>
 ```
 
+You can chain calls if you prefer as there's also a `stimulus_target` filter:
+
+```twig
+<div {{ stimulus_target('controller', 'a-target')|stimulus_target('other-controller', 'another-target') }}>
+    Hello
+</div>
+```
+
+You can also retrieve the generated attributes as an array, which can be helpful e.g. for forms:
+
+```twig
+{{ form_start(form, { 'attr': stimulus_target({
+    'controller': 'a-target',
+    'other-controller': 'another-target'
+).toArray() }) }}
```

Comment:
Same as above: use a single-controller example... and use this with a field instead of `form_start()` to be a bit more realistic